CRM/Financial/Form/Export.php

```diff
@@ -90,7 +90,11 @@ public function preProcess() {
     }
 
     $allBatchStatus = CRM_Core_PseudoConstant::get('CRM_Batch_DAO_Batch', 'status_id');
-    $this->_exportStatusId = CRM_Utils_Array::key('Exported', $allBatchStatus);
+    $this->_exportStatusId = civicrm_api3('OptionValue', 'getSingle', array(
```

Comment:
This is fine (although we should not merge until 4.7.30 rc is cut since we are keeping short-form arrays out until then) but a shortcut would be to use 'getvalue' rather than 'getsingle' - you would need to add the 'return' => 'value' param. 

Also, the 'sequential' => 1 param does nothing here - it causes the values to be re-indexed starting from 0 - but getsingle & getvalue return from a lower part of the array